common/culture/innovations/00_early_modern_innovations.txt

```diff
@@ -0,0 +1,221 @@
+﻿#### Local Types ####
+@administration_1 = "gfx/interface/icons/culture_innovations/innovation_administration_01.dds"
+@administration_2 = "gfx/interface/icons/culture_innovations/innovation_administration_02.dds"
```

Comment:
Do you plan to move the goblin tech to another era?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123693381-06ab7e00-d869-11eb-98a2-09f3b9e525e8.png)

</details>

Reply:
Which era did you have in mind?

Reply:
> Which era did you have in mind?

High Med?